Bugzilla/API/V1/Github.pm

```diff
@@ -423,4 +428,28 @@ sub _set_status_flag {
   }
 }
 
+# If the bug is being closed, then we also need to set the appropriate
+# nightly milestone version.
+sub _set_nightly_milestone {
+  my ($self, $bug) = @_;
+
+  # In order to determine the appropriate status flag for the 'master/main'
+  # branch, we have to find out what the current *nightly* Firefox version is.
+  # fetch_product_versions() calls an API endpoint maintained by rel-eng that
+  # returns all of the current product versions so we can use that.
+  my $versions = fetch_product_versions('firefox');
+  return if (!%$versions || !exists $versions->{FIREFOX_NIGHTLY});
+  my ($version) = split /[.]/, $versions->{FIREFOX_NIGHTLY};
+  return if !$version;
+
+  # Load the appropriate milestone.
+  my $value = "$version Branch";
+  my $milestone
+    = Bugzilla::Milestone->new({product => $bug->product_obj, name => $value});
```

Comment:
Out of scope for this change, but `Bugzilla::Milestone->new` should take some kind of `version` argument and do this `"$version Branch"` string interpolation in the function call instead of being a responsibility of the caller.